website/docs/getting-started/setup/instance-management/README.mdx

```diff
@@ -9,54 +9,63 @@ import TabItem from '@theme/TabItem';
 One of the key features of Appsmith is its instance management capabilities. This page is a gateway to various management tasks you can perform on your self-hosted instance. 
 
 You can manage your instance by:
-* Creating backups for the Appsmith instance and database
 * Updating Appsmith to a new version
 * Scheduling automatic updates for Appsmith installation
+* Creating backups for the Appsmith instance and database
 * Restoring backups
 
 This page provides information to set up and maintain your Appsmith installation.
 
 <div class="containerGridSampleApp">
```

Comment:
Let this be the old 2 cards per row as discussed. the last card will be in the third row